### PR TITLE
IJPS-121: Calculate Fee for a Medical member

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/IJPS-121

### Change description ###
Calculate Fee for a Medical member

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/jps/services/FeeServiceTest.java [hunk 1]
```diff
@@ -75,6 +79,36 @@ void shouldThrowIllegalArgumentExceptionWhenIdsMissing() {
...
+            "Judge",
+            LocalDate.now()
+        );
+
+        assertThat(fees.getId()).isEqualTo(id);
+    }
+
+    @Test
+    void shouldThrowIllegalArgumentExceptionWhenFeeMissing() {
+        LocalDate now = LocalDate.now();
```
Comment (markdathornehmcts) [inline src/test/java/uk/gov/hmcts/reform/jps/services/FeeServiceTest.java:103]: No need for local var here, use LocalDate.now directly

File: src/main/java/uk/gov/hmcts/reform/jps/services/PublishSittingRecordService.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            .map(flag -> fee.getLondonWeightedFee())
+            .orElse(fee.getStandardFee());
+    }
+
+    private BigDecimal getMedicalMemberFee(
+        String personalCode,
+        LocalDate sittingDate,
+        boolean higherMedicalRateSession,
+        Fee fee) {
+        BigDecimal derievedFee;
```
Comment (markdathornehmcts) [inline src/main/java/uk/gov/hmcts/reform/jps/services/PublishSittingRecordService.java:117]: name correction: derivedFee

File: src/test/java/uk/gov/hmcts/reform/jps/services/FeeServiceTest.java (hunk 1 above)
Comment (SomeshRA) [reply src/test/java/uk/gov/hmcts/reform/jps/services/FeeServiceTest.java:103]: Could have used LocalDate.now directly but sonar complains with code smell stating only invocation should be code under test in the assert block.

File: src/main/java/uk/gov/hmcts/reform/jps/services/PublishSittingRecordService.java (hunk 2 above)
Comment (SomeshRA) [reply src/main/java/uk/gov/hmcts/reform/jps/services/PublishSittingRecordService.java]: done

**Review by markdathornehmcts — APPROVED** [review]
LGTM

**Pull request merged**